docs/docs/building-modules/11-structure.md

```diff
@@ -53,6 +53,8 @@ x/{module_name}
 ├── module
 │   └── module.go
 │   └── abci.go
+├── integration
```

Comment:
is this accurate or are we keeping integration tests in `/test/integration` like in this PR?

Reply:
That's a left over from my previous trials. I will revert 👍